docs/ad/api.md

```diff
@@ -151,6 +138,133 @@ Options | Description | Type | Required
 }
 ```
 
+To set a category field for high cardinality:
+
+#### Request
+
+```json
+POST _opendistro/_anomaly_detection/detectors
+{
+  "name": "Second Test Detector 2",
+  "description": "ok rate",
+  "time_field": "@timestamp",
+  "indices": [
+    "host-cloudwatch"
+  ],
+  "category_field": [
+    "host"
+  ],
+  "feature_attributes": [
+    {
+      "feature_id": "latency_max",
+      "feature_name": "latency_max",
+      "feature_enabled": true,
+      "importance": 1,
+      "aggregation_query": {
+        "latency_max": {
+          "max": {
+            "field": "latency"
+          }
+        }
+      }
+    }
+  ],
+  "window_delay": {
+    "period": {
+      "interval": 10,
+      "unit": "SECONDS"
```

Comment:
Not sure if it's good to use `seconds` here as we only allow `Minutes` on AD Kibana.